CS410_Assignment2_CodeReview/src/sensormain/SensorApplication.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Lauren Kearney
+ * CS 410
+ * 
+ * three panel GUI for Sensor application
+ * 
+ * */
+
+package sensormain;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.awt.*;
+import javax.swing.*;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.border.TitledBorder;
+
+import sensor.PressureSensor;
+import sensor.RadiationSensor;
+import sensor.TemperatureSensor;
+
+public class SensorApplication extends JFrame {
+	
+	public PressureSensorAdapter pressure = new PressureSensorAdapter(new PressureSensor());
+	public RadiationSensorAdapter radiation = new RadiationSensorAdapter(new RadiationSensor());
+	public TemperatureSensorAdapter temperature = new TemperatureSensorAdapter(new TemperatureSensor());
+	
+	public SensorApplication() {
+		
+		int pressureLength = (int) (pressure.value());
+		pressureLength *= 10;
+		int radiationLength = (int) (radiation.value());
+		radiationLength *= 10;
+		int temperatureLength = (int) (temperature.value());
+		temperatureLength /= 4;
+		
+		
+		Dimension dimPressure = new Dimension();
+		dimPressure.height = 50;
+		dimPressure.width = pressureLength;
+		
+		Dimension dimRadiation = new Dimension();
+		dimRadiation.height = 50;
+		dimRadiation.width = radiationLength;
+		
+		Dimension dimTemperature = new Dimension();
+		dimTemperature.height = 50;
+		dimTemperature.width = temperatureLength;
+		
+		Dimension dimMax = new Dimension();
+		dimMax.height = 50;
+		dimMax.width = 100;
+		

```

Comment:
Recommend also setting a minimum dimension, if value is 0, there is no bar.
Optional design choice, looks like it could be missing to someone only seeing gui

<img width="712" alt="Screen Shot 2019-06-03 at 9 31 46 PM" src="https://user-images.githubusercontent.com/8973658/58851344-34e91000-8647-11e9-914c-3bb4b3b23fb3.png">